src/interfaces/octave/swig_typemaps.i

```diff
@@ -368,21 +368,50 @@ TYPEMAP_STRINGFEATURES_IN(is_matrix_type() && arg.is_uint16_type, uint16NDArray,
 
 /* output typemap for CStringFeatures */
 %define TYPEMAP_STRINGFEATURES_OUT(type,typecode)
+%typemap(out) shogun::SGStringList<char>
+{
+    /* TODO STRING OUT TYPEMAPS */
+    shogun::SGString<char>* str = $1.strings;
+    int32_t i, num_strings = $1.num_strings;
+
+    Cell c(num_strings, 1);
+
+    for (i = 0; i < num_strings; i++) {
+        c(i)=std::string(str[i].string);
+        SG_FREE(str[i].string);
```

Comment:
Does this maybe cause problems? The original string is destroyed?

Reply:
Correct. I copied these from the other languages interfaces. I removed them.